kubernetes/deployment-production.tmpl

```diff
@@ -50,6 +50,58 @@ spec:
           secret:
             secretName: seven-ten-production
 ---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: seven-ten-production-sidekiq
+  labels:
+    app: seven-ten-production-sidekiq
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: seven-ten-production-sidekiq
+  template:
+    metadata:
+      labels:
+        app: seven-ten-production-sidekiq
+    spec:
+      containers:
+        - name: seven-ten-production-sidekiq
+          image: zooniverse/seven-ten:__IMAGE_TAG__
```

Comment:
How will this image start the sidekiq service? Won't it start the puma app by default?

Reply:
Good catch. This will need an `args` option similar to Caesar https://github.com/zooniverse/caesar/blob/master/kubernetes/deployment-production.tmpl#L225

Reply:
```suggestion
          image: zooniverse/seven-ten:__IMAGE_TAG__
          args: ["/rails_app/docker/start_sidekiq.sh"]
```